.github/workflows/release.yml

```diff
@@ -0,0 +1,111 @@
+name: dbt-trino release
+
+on:
+  workflow_dispatch:
+
+# Release version number that must be updated for each release
+env:
+  version_number: "0.21.0"
```

Comment:
There are now several places to update before doing a release.

Could you please document throughly the release process for eventual new maintainers?

Reply:
I was looking at the pipeline again and I think that the whole approach in bumping version in here is wrong. 

Version bump is done just before doing a release to Github, and it's never tested.. tests needs to be done using new dbt version.

I need to redesign the release pipeline.

Reply:
I will describe the release process in README